startup.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-bundle install
+bundle check || bundle install
```

Comment:
Nice! Though is this change necessary? I don't think I've seen this pattern used elsewhere, and from memory, running `bundle install` when I have all the local dependencies already doesn't re-download/re-install the gems, it just exists quite quickly 🤔

Reply:
I borrowed it from here
https://github.com/alphagov/info-frontend/pull/891/commits/fe012d5e8120dab6145459fea888af2f6ee5325b
I believe that `check` is quicker than `install`, but it is probably a marginal gain for apps without a huge amount of dependencies. Happy for to remain or be removed!